src/main/java/com/airbnb/airpal/api/output/builders/CsvOutputBuilder.java

```diff
@@ -34,13 +35,13 @@
     @JsonIgnore
     private final UUID jobUUID;
 
-    public CsvOutputBuilder(boolean includeHeader, UUID jobUUID, long maxFileSizeBytes) throws IOException {
+    public CsvOutputBuilder(boolean includeHeader, UUID jobUUID, long maxFileSizeBytes, boolean compressedOutput) throws IOException {
         this.includeHeader = includeHeader;
         this.jobUUID = jobUUID;
         this.outputFile = File.createTempFile(jobUUID.toString(), FILE_SUFFIX);
         this.maxFileSizeBytes = maxFileSizeBytes;
         this.countingOutputStream = new CountingOutputStream(new FileOutputStream(this.outputFile));
-        this.csvWriter = new CSVWriter(new OutputStreamWriter(this.countingOutputStream));
+        this.csvWriter = new CSVWriter(new OutputStreamWriter(compressedOutput ? new GZIPOutputStream(this.countingOutputStream) : this.countingOutputStream));
```

Comment:
Please break this line up like described in https://github.com/airbnb/airpal/pull/160/files#r51830841